### PR TITLE
[release-4.14] OCPBUGS-26526: Documented to disable UWM telemetry writer in disconnected envs

### PR DESCRIPTION
**Pull request by jparrill**
This is a manual cherry-pick of https://github.com/openshift/hypershift/pull/3380

/assign jparrill

### CODE REVIEW

File: docs/content/how-to/disconnected/known-issues.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Additionally when you're provisioning the HostedCluster you will need to add a flag to indicate that the OLMCatalogPlacement is set to `Guest` because if that's not set, you will not be able to disable them.
+
+## Hypershift operator is failing to reconcile in Disconnected environments
+
+If you are operating in a disconnected environment and have deployed the Hypershift operator, you may encounter an issue with the UWM telemetry writer. Essentially, it exposes Openshift deployment data in your RedHat account, but this functionality does not operate in a disconnected environments.
+
+**Symptoms:**
+
+- The Hypershift operator appears to be running correctly in the `hypershift` namespace but even if you creates the Hosted Cluster nothing happens.
```
[the remark this answers is not in the document]
Comment (jparrill) [reply docs/content/how-to/disconnected/known-issues.md:20]: There is not a good way to infer if a HC should be disconnected or not, maybe we can think in a flag to remark this or something similar but after a discussion with the team before christmas shutdown, we decide that the best option for now it's just to document it.

**Comment by sjenning** [comment]
/override "Red Hat Trusted App Test / mce-24-txk4k / tektok-runner-enterprise-contract-mce-2.4"
/approve
/lgtm

**Pull request merged**